setup.cfg

```
@@ -14,27 +14,28 @@ classifiers =
     Development Status :: 4 - Beta
     Framework :: Django
     Framework :: Django :: 3.2
-    Framework :: Django :: 4.0
+    Framework :: Django :: 4.1
+    Framework :: Django :: 4.2
     Intended Audience :: Developers
     Operating System :: Unix
     Operating System :: MacOS
     Operating System :: Microsoft :: Windows
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
     Topic :: Software Development :: Libraries :: Python Modules
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    django>=3.1.0
+    Django >=3.2, <4.3
     django-better-admin-arrayfield
     django-solo
     glom
-    mozilla-django-oidc >=1.0.0, <3.0
+    mozilla-django-oidc >=2.0.0, <3.1.0
```

Same reasoning as above

```suggestion
    mozilla-django-oidc >=2.0.0
```

this may very well work out of the box with `mozilla-django-oidc` 3.1 when it releases, as that would be a backwards compatible release.